doc/itof/linux_build_instructions.md

```diff
@@ -75,3 +75,23 @@ make
 ## SDK with bindings
 
 - Please check the readme files for each type of binding in the [bindings directory](../../bindings).
+
+## Generate doxygen documentation
+
+Requirements:
+* Doxygen
+* Graphviz
+
+```console
+sudo apt-get install doxygen doxygen-doc doxygen-gui graphviz
```

Comment:
is doxygen-gui required?
is doxygen-doc required?

Reply:
Yes, they are not required.